scipy/sparse/_array_api/__init__.py

```diff
@@ -0,0 +1,196 @@
+from .utils import fill_array_with_not_implemented, fill_linalg_with_not_implemented, fill_array_module_with_not_implemented
+
+from .csr import *
+from ..import linalg
+from numpy.array_api import(
```

Comment:
`numpy.array_api` is now `array_api_strict`. Why can't we get the dtypes from `numpy`, though?